### ISSUE TITLE
Make some contact variables editable from glific interface

### ISSUE THREAD

**Issue by abhi1203** [glific/glific-frontend#2274]
**Is your feature request related to a problem? Please describe.**
Sometimes when a user has entered an incorrect information, NGOs would like to make the correct right there from the interface itself.

**Describe the solution you'd like**
From the contact profile section, the fields should be editable.

**Comment by mdshamoon** [comment]
I don't think we have an API  to update contact variable directly for a contact. Moving the issue to the backend repo.
